### PR TITLE
[DOC] Add badges to the change log

### PR DESCRIPTION
**Pull request by ymzayek**
Supersedes and closes #3280.

Opinions encouraged on which labels to use!
For now I just put API, Plotting, Doc, Deprecation, Test

I'm using inline badges formatted by sphinx design, from which our theme comes (https://sphinx-design.readthedocs.io/en/furo-theme/badges_buttons.html). It seems possible to create custom ones however I wouldn't do this unless we feel it's necessary to cover all desired labels. For now, what's there seems good and plenty enough.

Thanks @NicolasGensollen for initiating this!

### CODE REVIEW

File: doc/changes/0.10.0.rst [hunk 1]
```diff
@@ -39,65 +39,65 @@ HIGHLIGHTS
...
+- :bdg-info:`Plotting` Fix :func:`~plotting.plot_connectome` which was raising a ``ValueError`` when ``vmax < 0`` (:gh:`3390` by `Paul Bogdan`_).
 
-- Change the order of applying ``sample_masks`` in :func:`~signal.clean` based on different filtering options (:gh:`3385` by `Hao-Ting Wang`_).
+- :bdg-success:`API` Change the order of applying ``sample_masks`` in :func:`~signal.clean` based on different filtering options (:gh:`3385` by `Hao-Ting Wang`_).
 
-- When using cosine filter and ``sample_masks`` is used, :func:`~signal.clean` generates the cosine discrete regressors using the full time series (:gh:`3385` by `Hao-Ting Wang`_).
+- :bdg-success:`API` When using cosine filter and ``sample_masks`` is used, :func:`~signal.clean` generates the cosine discrete regressors using the full time series (:gh:`3385` by `Hao-Ting Wang`_).
 
-- Description of the ``opening`` parameter added to ``compute_multi_epi_mask`` docs (:gh:`3412` by `Natasha Clarke`_).
+- :bdg-primary:`DOC` Description of the ``opening`` parameter added to ``compute_multi_epi_mask`` docs (:gh:`3412` by `Natasha Clarke`_).
```
Comment (tsalo) [inline doc/changes/0.10.0.rst:58]: Maybe Doc instead of DOC, since it's not an acronym/initialism.

File: doc/changes/0.10.0.rst [hunk 2]
```diff
@@ -39,65 +39,65 @@ HIGHLIGHTS
...
-- Description of the ``opening`` parameter added to ``compute_multi_epi_mask`` docs (:gh:`3412` by `Natasha Clarke`_).
+- :bdg-primary:`Doc` Description of the ``opening`` parameter added to ``compute_multi_epi_mask`` docs (:gh:`3412` by `Natasha Clarke`_).
 
-- Fix display of colorbar in matrix plots. Colorbar was overlapping in :func:`~matplotlib.pyplot.subplots` due to a hardcoded adjustment value in the subplot (:gh:`3403` by `Raphael Meudec`_).
+- :bdg-info:`Plotting` Fix display of colorbar in matrix plots. Colorbar was overlapping in :func:`~matplotlib.pyplot.subplots` due to a hardcoded adjustment value in the subplot (:gh:`3403` by `Raphael Meudec`_).
 
-- Pass values with correct type to scikit-learn estimator parameters and remove deprecated parameter (:gh:`3430` by `Yasmin Mzayek`_).
+- :bdg-success:`API` Pass values with correct type to scikit-learn estimator parameters and remove deprecated parameter (:gh:`3430` by `Yasmin Mzayek`_).
 
 - Fix cluster labeling for two-sided cluster level permutation tests in :func:`~mass_univariate.permuted_ols` and associated function test ``test_cluster_level_parameters_smoke`` (:gh:`3436` by `Jelle Roelof Dalenberg`_).
```
Comment (bthirion) [inline doc/changes/0.10.0.rst:64]: No badge here ?

File: doc/changes/latest.rst [hunk 3]
```diff
@@ -31,22 +31,23 @@ Fixes
...
+
+- :bdg-success:`API` Allow users to output label maps from :func:`~reporting.get_clusters_table` (:gh:`3477` by `Steven Meisler`_).
 
 Changes
 -------
 
-- The documentation for :func:`~image.threshold_img` has been improved, with more information about which voxels are set to 0 and which ones keep their original values (:gh:`3485` by `Rémi Gau`_).
+- :bdg-primary:`Doc` The documentation for :func:`~image.threshold_img` has been improved, with more information about which voxels are set to 0 and which ones keep their original values (:gh:`3485` by `Rémi Gau`_).
 
 - Modules :mod:`~nilearn.decomposition` and :mod:`~nilearn.decoding` code and docstrings have been reformatted using black. Changes resulted in improved readability overall and increased consistency (:gh:`3491` and :gh:`3484` by `Rémi Gau`_).
```
Comment (bthirion) [inline doc/changes/latest.rst:53]: and here ?

**Comment by ymzayek** [comment]
Badge mapping so far (subject to change):

- :bdg-primary:\`Doc\`
- :bdg-secondary:\`Maint\`
- :bdg-success:\`API\`
- :bdg-info:\`Plotting\`
- :bdg-warning:\`Test\`
- :bdg-danger:\`Deprecation\`
- :bdg-light:\`Code\`

Not assigned:

- :bdg-dark:``

File: doc/changes/0.10.0.rst (hunk 2 above)
Comment (ymzayek) [reply doc/changes/0.10.0.rst]: Added now

File: doc/changes/latest.rst (hunk 3 above)
Comment (ymzayek) [reply doc/changes/latest.rst]: Same

**Comment by ymzayek** [comment]
I think I might apply them backwards just to v0.9, and then open a PR that would be a `Good first issue` for contributors to apply them on the other versions. If that sounds good to everybody

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Comment by ymzayek** [comment]
This one is ready. Once merged I'll open an issue to apply the badges backwards to the rest of the changelog.

**Review by Remi-Gau — APPROVED** [review]

**Comment by ymzayek** [comment]
@Remi-Gau I would say this is something that maintainers can do before a release (moving forward). In that case we can document it in the maintenance docs in the [making a release section](https://nilearn.github.io/stable/maintenance.html#how-to-make-a-release).

For finishing applying it backwards, I'll open an issue.

WDYT?

**Comment by Remi-Gau** [comment]
> @Remi-Gau I would say this is something that maintainers can do before a release (moving forward). In that case we can document it in the maintenance docs in the [making a release section](https://nilearn.github.io/stable/maintenance.html#how-to-make-a-release).
> 
> For finishing applying it backwards, I'll open an issue.
> 
> WDYT?

yup that's good with me

**Comment by bthirion** [comment]
Merging, then.

**Pull request merged**